datalad/crawler/nodes/misc.py

```diff
@@ -589,3 +591,15 @@ def __call__(self, data):
             lgr.info("Ran node %s which yielded %d times", node, n)
             pdb.set_trace()
 
+
+def fix_url(data):
+    """Given data, get value within 'url' key and fix up so it is legit url
+
+    - replace spaces with %20
+    """
+
+    for key, value in data.items():
```

Comment:
I left a comment before -- please review (it should have come in email as well):  we do not want to process all `data` values, only `"url"` (for now).  To make it more generic, add a keyword argument to this function (`keys=["url"]) and add tests where you change it so "it does the right thing"